### ISSUE TITLE
Bug: Cannot move selection below table

### ISSUE THREAD

**Issue by Zombobot1** [facebook/lexical#2010]
Lexical version: lexical playground

## Steps To Reproduce

1. insert a table inside existing text
2. start selection above table and move it down using arrows

## The current behavior
Selection gets stuck at the table bottom, instead of moving to the next line

## The expected behavior
Selection moves to the next line below the table

![image](https://user-images.githubusercontent.com/48030070/165650237-79f2a04a-ee35-481d-850e-aaf84e46635c.png)

**Comment by tylerjbainbridge** [comment]
@Zombobot1 is this still happening for you? On the current [Lexical Playground](https://playground.lexical.dev/) I can move the selection from the text above, into the table, and then below.

**Comment by Zombobot1** [comment]
Yes, I also can move it down to the line "If you'd like to find out more about Lexical, you can"
However, afterwards I cannot move it back up. It gets stuck in the table
